### DISCUSSION TITLE
Twitter Oauth meta object username?

### DISCUSSION THREAD

**Discussion by janzheng** [pocketbase/pocketbase#385] [Q&A] [answered]
Hi everyone,

I've been noodling around with Twitter Oauth and the user object you get back via `authViaOAuth2()` looks like:
```
  meta: {
    id: '1022157402014195712',
    name: 'Jan "yawn" (🥲,🥲 )',
    email: '',
    avatarUrl: 'https://pbs.twimg.com/profile_images/1380947684610166787/ZQNanc9D_normal.jpg'
  },
```

Is it possible to get the person's twitter username? Is that a setting on the Twitter dashboard, something implemented on the app side, or something on Pocketbase's side? Or is it just impossible to get?

**Comment by ganigeorgiev** [comment, marked as answer]
It is possible to get the Twitter account username but we will have to request it from the Twitter `/2/users/me` API and add a new field to the [`AuthUser`](https://github.com/pocketbase/pocketbase/blob/master/tools/auth/auth.go) structure.

_I'll consider adding it in the next minor release v0.6.1 planned sometime this week._

**Reply by janzheng** [reply]
oh thank you! No hurry, just wondering if it's something that I missed on my end. Love all the latest changes to Pocketbase!!

**Reply by ganigeorgiev** [reply]
This is now available as part of [v0.7.0 release](https://github.com/pocketbase/pocketbase/releases/tag/v0.7.0)
